testing/integrations_test.go

```diff
@@ -174,6 +174,51 @@ func TestPrivateRedmineIntegration(t *testing.T) {
 	}
 }
 
+func TestPublicAzureIntegration(t *testing.T) {
+	err := scenariobuilder.NewScenario().
+		OnlyRunOnCI().
+		WithBinary("../todocheck").
+		WithBasepath("./scenarios/integrations/azureboards_public").
+		WithConfig("./test_configs/integrations/azureboards_public.yaml").
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeMalformed).
+				WithLocation("./scenarios/integrations/azureboards_public/main.go", 3).
+				ExpectLine("// TODO: 1 A malformed issue")).
+		ExpectTodoErr(
+			scenariobuilder.NewTodoErr().
+				WithType(errors.TODOErrTypeNonExistentIssue).
+				WithLocation("./scenarios/integrations/azureboards_public/main.go", 7).
+				ExpectLine("// TODO 999: A non-existent issue")).
+		Run()
+	if err != nil {
+		t.Errorf("%s", err)
+	}
+}
+
+func TestPrivateAzureIntegration(t *testing.T) {
```

Comment:
The integration test doesn't handle the line "// TODO 6: An issue in CLOSED column" in the testing scenarios as closed issue error.